runtime-modules/governance/src/council.rs

```diff
@@ -60,6 +66,14 @@ impl<T: Trait> Module<T> {
     pub fn is_councilor(sender: &T::AccountId) -> bool {
         Self::active_council().iter().any(|c| c.member == *sender)
     }
+
+    // Initializes a new mint
+    pub fn create_new_council_mint() -> T::MintId {
+        let mint_id = <minting::Module<T>>::add_mint(minting::BalanceOf::<T>::zero(), None)
+            .expect("Failed to create a mint for the council");
```

Comment:
I'm not happy with panic in the runtime. Can we put mint only on successful creation and during the setting mint parameters - introduce another "ensure_*"? check on whether mint actually exist?

Reply:
That is a fair point. I originally had this `expect()` inside the `build()` method before refactoring, where it was okay to panic while constructing genesis. 

Given that add_mint can fail and we call it in the migration, I will have to make the mint an `Option<MintId>`

Let me revise.

Reply:
Updated to make the mint an `Option` so it doesn't need to be created at genesis and we can avoid panic. The mint can be automatically created when setting capacity if it doesn't exist.